gradle.properties

```diff
@@ -7,7 +7,9 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 128
+
```

Comment:
Slightly nit picky: Can you remove the extra whitespace added on line 10 + 11. As everyone in the platform team regularly edits this line.